src/cleanvision/imagelab.py

```diff
@@ -211,6 +314,42 @@ def report(
         examples_per_issue: Optional[int] = None,
         verbosity: int = 1,
     ) -> None:
+        """Prints a summary of issues found in the dataset with their example images from the dataset.
+        By default, if no arguments are specified, it reports the top issues found in the dataset.
+
+        Parameters
+        ----------
+        issue_types : List[str], optional
+            List of issue types to report
+
+        num_top_issues : int, default=3
+            Number of top issues to report. It is ignored if issue_types is given
+
+        max_prevalence : float, default=0.5
+            Between 0 and 1.
+            Ignores the issue types from reporting if found in more than max_prevalence fraction of the dataset.
+            It is ignored if issue_types is given.
```

Comment:
```suggestion
            Value between 0 and 1, ignored if `issue_types` is provided. 
            Otherwise issue types that are detected in more than `max_prevalence` fraction of the images in dataset will be omitted from report.
            You are presumably already aware of these in your dataset.
```